src/components/format-bytes/format-bytes.test.ts

```diff
@@ -0,0 +1,167 @@
+import { expect, fixture, html, elementUpdated } from '@open-wc/testing';
+import type SlFormatBytes from './format-bytes';
+
+describe('<sl-format-bytes>', () => {
+
+  describe('defaults ', () => {
+
+    it('default properties', async () => {
+      const el = await fixture<SlFormatBytes>(html` <sl-format-bytes></sl-format-bytes> `);
+
+      expect(el.value).to.equal(0);
+      expect(el.unit).to.equal('byte');
+      expect(el.display).to.equal('short');
+      expect(el.lang).to.be.undefined;
+    });
+  });
+
+  describe('bytes', () => {
+
+    it('bytes : short display', async () => {
+      const el = await fixture<SlFormatBytes>(html` <sl-format-bytes></sl-format-bytes> `);
+
+      el.value = 12;
+      await elementUpdated(el);
+      // webkit keeps the lit comment in textContent
+      expect(el.shadowRoot?.textContent?.endsWith('12 byte')).to.be.true;
```

Comment:
Weird that this happens in WebKit but not other browsers. Is there a cleaner way to test this? Using `endsWith()` could lead to inconsistent results, especially in cases like `'212 byte'.endsWith('12 byte')`.

(Aside, I wish the browser would output "bytes" here, but standards!)

Reply:
i figured it out. might have been a cached test issue because i could have sworn i started with `textContent`, but `textContent` works